tests/Unit/Core/ResponseTest.php

```
@@ -253,6 +253,21 @@ public function testSendHtmlOutputsHtmlBody(): void
         http_response_code($previousStatus ?: 200);
     }
 
+    #[Test]
+    public function testSetStatusDoesNotEmitBody(): void
+    {
+        $previousStatus = http_response_code();
+
+        ob_start();
+        Response::setStatus(403);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Core\Response' in method 'testSetStatusDoesNotEmitBody'. (undefined)

(StaticAccess)